Translation corrections (#108)
07th-mod/higurashi-console-arcs: Update/some_009.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300489", 256, TRUE);
	OutputLine(NULL, "「元々、佐伯病院の師長さんまで勤めていた優秀な看護師さんでね。",
			NULL, "\"She was an excellent nurse. She even used to be the head nurse at Saeki Hospital.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300490", 256, TRUE);
	OutputLine(NULL, "あの介護福祉センターの設立を最初に提案したひとりだったんだよ」",

> 			NULL, " It was her who originally proposed establishing the care and welfare center.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " She was the one who originally proposed establishing the care and welfare center.\"", Line_Normal);
```

	ClearMessage();

//rでも、…師長に昇格してからの山瀬さんの仕事は、それまでにも増して忙しくなった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "でも、…師長に昇格してからの山瀬さんの仕事は、それまでにも増して忙しくなった。",
			NULL, "But... her workload got even heavier after she was promoted to head nurse.", Line_Normal);
	ClearMessage();

//rそもそも看護師さんは２４時間で患者さんを診て回るために不規則な時間帯シフトで働く職業で、その管理職ともなれば昼夜どころか、休みすらも満足に過ごせないことが多い。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そもそも看護師さんは２４時間で患者さんを診て回るために不規則な時間帯シフトで働く職業で、その管理職ともなれば昼夜どころか、休みすらも満足に過ごせないことが多い。",
			NULL, "Nursing is already the kind of profession where you have to work irregular shifts to provide patients with 24 hour care, and being a manager on top of that leaves you with little to no free time.", Line_Normal);
	ClearMessage();

//r…そのために、家庭内のことに対しても手を回しきれず、お父さんの世話はご主人、そして息子さんにも手伝ってもらうことにしたそうだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…そのために、家庭内のことに対しても手を回しきれず、お父さんの世話はご主人、そして息子さんにも手伝ってもらうことにしたそうだ。",

> 			NULL, "...For that reason, she couldn't do much at home, so her husband and son decided to help take care of her, but I heard that her son decided to help out with taking care of her father.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...For that reason, she couldn't do much at home, so her husband and son decided to help take care of her father.", Line_Normal);
```

			NULL, "\"...Yamase-san never blamed her son.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300492", 256, TRUE);
	OutputLine(NULL, "むしろ、その時の症状についてどうしてちゃんと教えておかなかったんだ、って後悔し続けてね。",

> 			NULL, " Rather, she regretted not properly teaching him about the symptoms her father had displayed the time.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Rather, she regretted not properly teaching him about the symptoms her father had displayed at the time.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300494", 256, TRUE);
	OutputLine(NULL, "「…そう言って、介護福祉センターの設立を私のお父さんに提案して、お年寄りの世話とリハビリを行える施設を立ち上げたんだ」",
			NULL, "\"...With that, she proposed the establishment of the care and welfare center to my father, and they built a facility to care and rehabilitate the elderly.\"", Line_Normal);
	ClearMessage();

//千紗登rvS04/37/310300495.「そして、介護士って仕事を医療の人たちにも認知してもらって、介護の重要性をいろんな人に理解してもらうために毎日、頑張っていたのにっ…！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300495", 256, TRUE);
	OutputLine(NULL, "「そして、介護士って仕事を医療の人たちにも認知してもらって、介護の重要性をいろんな人に理解してもらうために毎日、頑張っていたのにっ…！」",
			NULL, "\"And she worked hard every day so those who worked in and out of medicine would acknowledge and understand the contributions and importance of caregivers! So why...!?\"", Line_Normal);
	ClearMessage();

//rそこまで話して、…千紗登ちゃんの奥歯がぎりっ、と鳴る。硬く握り締められた両手のこぶしは、爪が食い込むほどに真っ白になっていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そこまで話して、…千紗登ちゃんの奥歯がぎりっ、と鳴る。硬く握り締められた両手のこぶしは、爪が食い込むほどに真っ白になっていた。",

> 			NULL, "Chisato-chan ground the back of her teeth at that point. She clenched her fists so tight that they were turning white, and her nails began to dig in.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Chisato-chan grit her teeth at that point. She clenched her fists so tight that they were turning white, and her nails began to dig in.", Line_Normal);
```

	ClearMessage();

//千紗登rvS04/37/310300496.「こんな惨いかたちで、死ぬような人じゃなかったんだっ！kvS04/37/310300497.　もっともっとたくさんの人のために働いて、頑張って！kvS04/37/310300498.　色んな人に認められて幸せになる資格があった！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300496", 256, TRUE);
	OutputLine(NULL, "「こんな惨いかたちで、死ぬような人じゃなかったんだっ！",
			NULL, "\"She didn't deserve to die so horribly!", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300497", 256, TRUE);
	OutputLine(NULL, "　もっともっとたくさんの人のために働いて、頑張って！",

> 			NULL, " She worked her hardes for so many people!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " She worked her hardest for so many people!", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300501", 256, TRUE);
	OutputLine(NULL, "「おまけに、最後は自分で喉引っ掻いて死んだっ？！",
			NULL, "\"And then he clawed out his throat and died anyway?!", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300502", 256, TRUE);
	OutputLine(NULL, "　ふざけんな！",
			NULL, " Screw that!", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300503", 256, TRUE);
	OutputLine(NULL, "　死にたきゃ一人で首くくれっ！！",
			NULL, " If he wanted to die so bad, he should have just killed himself and left it at that!!", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300504", 256, TRUE);
	OutputLine(NULL, "　あんなに毎日頑張ってた山瀬さんを殺して！",

> 			NULL, " But instead, he went and killed a hardworker Yamase-san on top of that!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " But instead, he went and killed hardworking Yamase-san on top of that!", Line_WaitForInput);
```
